environment_setup/setup_software.sh

```diff
@@ -58,6 +58,7 @@ host_software_packages=(
     python3-yaml # Load dynamic parameter configuration files
     qt5-default # The GUI library for our visualizer
     valgrind # Checks for memory leaks
+    libsqlite3-dev # needed to build Python 3 with sqlite support
```

Comment:
Do we need sqlite support? Db for ML/proto data?

Reply:
Having the built-in sqlite is a dependency of Jupyter Notebook